libs/wire-api/src/Wire/API/Event/FeatureConfig.hs

```diff
@@ -52,7 +52,6 @@ instance ToSchema EventType where
 data EventData
   = EdFeatureWithoutConfigChanged TeamFeatureStatusNoConfig
   | EdFeatureApplockChanged (TeamFeatureStatusWithConfig TeamFeatureAppLockConfig)
```

Comment:
Removing `TeamFeatureClassifiedDomainsConfig` causes build to fail due to unused imports.